server/oauth2.go

```diff
@@ -245,6 +245,8 @@ type idTokenClaims struct {
 	Groups []string `json:"groups,omitempty"`
 
 	Name string `json:"name,omitempty"`
+
+	Login string `json:"dex.coreos.com/login"`
```

Comment:
e.g. these are the claims kubernetes uses https://github.com/kubernetes/kubernetes/blob/v1.6.0-beta.4/pkg/serviceaccount/jwt.go#L44-L47